kernel/src/main/java/org/kframework/kompile/KompileOptions.java

```diff
@@ -38,6 +39,9 @@ GlobalOptions getGlobalOptions_UseOnlyInGuiceProvider() {
     @ParametersDelegate
     public OuterParsingOptions outerParsing = new OuterParsingOptions();
 
+    @ParametersDelegate
+    public transient OutputDirectoryOptions outputDirectory = new OutputDirectoryOptions();
+
```

Comment:
I'm a bit confused. Wasn't OuterParsingOptions supposed to be transient, and OutputDirectoryOptions be serializable?